src/style.css

```
@@ -1,3 +1,80 @@
+:root {
+  --blue-color: #00acdc;
+  --black-color: #000;
+  --padding-size: 50px;
+}
+
+* {
+  margin: 0;
+  padding-inline-start: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+}
+
+.spacer {
+  flex-grow: 1;
+}
+
+header {
+  padding-left: var(--padding-size);
+  padding-right: var(--padding-size);
```

You could use a shorter form for this
```suggestion
  padding: 0 var(--padding-size);
```